enzyme/Enzyme/ActivityAnalysis.cpp

```diff
@@ -730,6 +731,9 @@ bool isPossibleFloat(const TypeResults &TR, Value *I, const DataLayout &DL) {
 /// do not propagate adjoints themselves
 bool ActivityAnalyzer::isConstantInstruction(TypeResults const &TR,
                                              Instruction *I) {
+
+  TimeTraceScope timeScope("is constant instruction");
```

Comment:
Can you change this to "ActivityAnalysis (isConstantInstruction)", I->getParent()->getParent()->getName(), I->getName())

Reply:
eh in this case let's just use the last one for scoping reasons